src/gen/lib/Codegen_AST.ml

```diff
@@ -20,6 +20,7 @@ let preamble grammar =
    entrypoint: %s
 *)
 
+open! Sexplib.Conv
```

Comment:
you need the !?

Reply:
I guess I'm doing something wrong. The `open` shouldn't be needed at all.

There's also the problem that the sexp representation loses information about the different kinds of types (variant, tuple, list, option) and I don't want to invest too much time in this. If a better dumper turns out to be needed, we can generate our own code for pretty-printing an AST... This would convert a grammar-specific AST to a generic tree of this type:
```
type t =
| Atom of string
| List of t list
| Variant of string * t list option
| Option of t option
| Tuple of t list
```
and then we'd pretty-print this however we like.